### PR TITLE
Feature/#3

### PR DESCRIPTION
**Pull request by LTiger14**

### CODE REVIEW

File: src/group-imports.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        statementGroupingText += current[x];
+        break;
+      } else {
+        break;
+      }
+    }
+
+    if (last) {
+      if (last !== statementGroupingText) {
+        const blankLinePostion = new vscode.Position(currentImports[currentImports.length - 1].endPosition.line + 1, 1);
```
Comment (znikola) [inline src/group-imports.ts]: In #6, I fixed the startPosition and endPosition to start from 1, instead from 0.

File: src/group-imports.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        currentImports.push(imports[i]);
+      }
+    } else {
+      currentImports.push(imports[i]);
+    }
+
+    last = statementGroupingText;
+
+    if (i === imports.length - 1) {
+      const blankLinePostion = new vscode.Position(currentImports[currentImports.length - 1].endPosition.line + 1, 1);
```
Comment (znikola) [inline src/group-imports.ts]: The same comment as on line 31

File: src/models/import-group.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+import { Import } from './import';
+import { Position } from 'vscode';
```
Comment (znikola) [inline src/models/import-group.ts]: This import statement should go first

Comment (LTiger14) [reply src/models/import-group.ts]: 😂 Import sorting extension with it's import not sorted haha

**Pull request merged**